caps/types.go

```diff
@@ -20,66 +20,76 @@
 package caps
 
 import (
-	"encoding/json"
 	"fmt"
 )
 
 // Type describes a group of interchangeable capabilities with common features.
 // Types are managed centrally and act as a contract between system builders,
 // application developers and end users.
-type Type struct {
-	// Name is a key that identifies the capability type. It must be unique
-	// within the whole OS. The name forms a part of the stable system API.
-	Name string
-	// RequiredAttrs contains names of attributes that are required by
-	// capability of this type.
-	RequiredAttrs []string
+type Type interface {
+	// Unique and public name of this type.
+	Name() string
+	// Sanitize a capability (altering if necessary).
+	Sanitize(c *Capability) error
```

Comment:
I'm curious, why was the name changed to Sanitize? That is something you usually do on a toilet to clean it (and in the hospital of course). I'm not a native speaker so I could be wrong and this is bike-shed territory for sure. What exactly will this function do? What does "altering if necessary" means? What kind of alteration?